rfhub2/db/repository/keyword_repository.py

```diff
@@ -36,9 +36,17 @@ def get_all(
                 ]
         if collection_name:
             filter_criteria.append(Collection.name.ilike(glob_to_sql(collection_name)))
+        keyword_statistics = self.session.query(func.sum(Statistics.times_used).label('times_used'),
+                                                (func.sum(Statistics.total_elapsed_time) / func.sum(
+                                                    Statistics.times_used)).label('avg_elapsed_time'), Statistics.collection,
+                                                Statistics.keyword).group_by(Statistics.collection,
+                                                                             Statistics.keyword).subquery()
         return (
-            self.session.query(Keyword)
+            self.session.query(Keyword, Collection.name, Collection.id, keyword_statistics.c.times_used.label('times_used'), keyword_statistics.c.avg_elapsed_time.label('avg_elapsed_time'))
             .join(Keyword.collection)
+            # .join(Collection)
+            .outerjoin(keyword_statistics, and_(Collection.name == keyword_statistics.c.collection,
+                                                Keyword.name == keyword_statistics.c.keyword))
             .filter(*filter_criteria)
```

Comment:
@pbylicki I'm totally stuck at this one. Everything is working fine until I add needed columns to query. When I do, mapping screams with 
```
response -> 99 -> collection
  field required (type=value_error.missing)
```
Any suggestion